Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -73,6 +77,7 @@ try {
     $artifact = $repo.artifact
     $installApps = $repo.installApps
     $installTestApps = $repo.installTestApps
+    $doNotBuildTests = $repo.doNotBuildTests

```

Comment:
might be easier to read and avoiding double negative by consider using positive flags in this case BuildTests

Reply:
Could be - will think about that one for a refactor task